Common/include/basic_types/ad_structure.hpp

```diff
@@ -676,6 +719,42 @@ FORCEINLINE void ResumePreaccumulation(bool wasActive) {
   SU2_OMP_SAFE_GLOBAL_ACCESS(PreaccEnabled = true;)
 }
 
+struct ErrorReport {
+  unsigned long ErrorCounter;
+  std::ofstream* out;
```

Comment:
Worth having default initialization?
```suggestion
  unsigned long ErrorCounter = 0;
  std::ofstream* out = &std::cout;
```

Reply:
The one I suggested

Reply:
Then use the base type for cout, cerr, ofstream, etc.